slice/Ice/Locator.ice

```diff
@@ -113,7 +113,7 @@ module Ice
     }
 
     /// This interface should be implemented by services implementing the <code>Ice::Locator interface</code>. It should
-    /// be advertised through an Ice object with the identity <code>`Ice/LocatorFinder'</code>. This allows clients to
+    /// be advertised through an Ice object with the identity <code>'Ice/LocatorFinder'</code>. This allows clients to
```

Comment:
Which syntax should be use in doc-comments to insert a string as is? `<code>text</code>` or back ticks?

Reply:
Doxygen seems to support both HTML `<code>` tags and Markdown backticks:
https://www.doxygen.nl/manual/htmlcmds.html
https://www.doxygen.nl/manual/markdown.html#md_codespan

I'm not sure if there's some subtle difference between them, but obviously we should prefer backticks over xml tags if there's no difference.